controlplane/kubeadm/internal/controllers/controller_test.go

```diff
@@ -1385,7 +1385,7 @@ kubernetesVersion: metav1.16.1
 		g.Expect(kcp.Finalizers).To(ContainElement(controlplanev1.KubeadmControlPlaneFinalizer))
 
 		g.Expect(kcp.Status.Selector).NotTo(BeEmpty())
-		g.Expect(kcp.Status.Replicas).To(BeEquivalentTo(1))
+		g.Expect(ptr.Deref(kcp.Status.Replicas, 0)).To(BeEquivalentTo(1))
```

Comment:
Nit: You could also use `HaveValue` and avoid the `ptr.Deref`. That would tell us whether the pointer was nil or not, which the current version of this will not

Reply:
Thx, done